cinq_auditor_iam/__init__.py

```diff
@@ -53,13 +54,36 @@ def run(self, *args, **kwargs):
             Account.enabled == 1,
             Account.account_type == AccountTypes.AWS
         )
+        self.manage_policies(accounts)
+        self.update_role_timeouts(accounts)
+
+    def update_role_timeouts(self, accounts):
+        if not accounts:
+            return
+        timeout_in_seconds = self.dbconfig.get('role_timeout_in_hours', self.ns, 8) * 60 * 60
+        for account in accounts:
+            sess = get_aws_session(account)
+            iam = sess.client('iam')
+            try:
+                role_list = iam.list_roles()['Roles']
+                for role in role_list:
+                    self.log.info('Attempting to adjust MaxSessionDuration for role {} in account {}'.format(role['RoleName'], account.account_name))
```

Comment:
With the try/except moved, we can remove this log line, since we will always log failures or successful changes